mailbox/store/src/main/java/org/apache/james/mailbox/store/search/comparator/SentDateComparator.java

```diff
@@ -34,27 +32,35 @@
 public class SentDateComparator extends AbstractHeaderComparator {
     public final static Comparator<MailboxMessage> SENTDATE = new SentDateComparator();
 
+    private SentDateComparator() {
+        super();
+    }
+
     @Override
-    public int compare(MailboxMessage o1, MailboxMessage o2) {
-        Date date1 = getSentDate(o1);
-        Date date2 = getSentDate(o2);
-        int i = date1.compareTo(date2);
+    public int compare(MailboxMessage message1, MailboxMessage message2) {
+        Date date1 = getSentDateOrInternalDate(message1);
+        Date date2 = getSentDateOrInternalDate(message2);
+        int diff = date1.compareTo(date2);
```

Comment:
Yes but every one understand diff, by the way I am not even sure to know the full english word